tests/cases/fourslash/stringCompletionsUnterminated.ts

```diff
@@ -0,0 +1,36 @@
+///<reference path="fourslash.ts" />
+
+// @Filename: file1.ts
+//// const a = { "foo.bar": 1 }
+//// a["[|foo.b/*0*/|]
+//// a["[|foo.b           /*1*/|]
+//// a[    "[|foo.b/*2*/|]
+//// a["[|foo.b/*3*/|]"
+//// a["[|foo./*4*/b|]
+//// a['[|foo./*5*/b|]
+//// a[`[|foo./*6*/b|]

```

Comment:
Can you add test cases for when you have `foo.` (no contents following the dot) and `fo` (no dot or following contents)?

Reply:
added! I also tested manually in vscode to see that the behavior didn't change for the previously working cases even though it does return a `replacementSpan` now.